### ISSUE TITLE
Add DM data files

### ISSUE THREAD

**Issue by cdeil** [gammapy/gammapy#1557]
@joleroi - I tried to run [astro_dark_matter.ipynb](https://github.com/gammapy/gammapy-extra/blob/master/notebooks/astro_dark_matter.ipynb) and it fails because 
```
$GAMMAPY_EXTRA/datasets/dark_matter_spectra/AtProduction_gammas.dat
```
isn't there:
https://github.com/gammapy/gammapy-extra/tree/master/datasets

This is mentioned here:
https://github.com/gammapy/gammapy/blob/7abc4bacecb4b36270b59a975fa9649144c76750/gammapy/astro/darkmatter/spectra.py#L42

Can you please add any data files needed and also a README in that folder where the data comes from?

**Comment by cdeil** [comment]
@joleroi - I just now did a small fix in master for `darkmatter`: 0cd7cd83b36bd46356ce22df9c786ca017c29f2f

I noticed that it has zero tests!?

It does use `ParameterList`, which I wanted to refactor today; you also use things from `gammapy.maps` which we want to refactor. Zero tests is really problematic.

Please add a few simple unit tests that execute the key functionality once and have one assert at the end.

**Comment by joleroi** [comment]
Solved in https://github.com/gammapy/gammapy-extra/commit/56e4baf4e30b61ddcb38a5f8abd50f0d39c882c7
